laravel/app/Console/Kernel.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Console;
+
+use App\Console\Commands\StatsCleanup;
+use App\Console\Commands\StatsDownload;
+use Illuminate\Console\Scheduling\Schedule;
+use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
+use Illuminate\Support\Facades\Artisan;
+
+class Kernel extends ConsoleKernel
+{
+    /**
+     * The Artisan commands provided by your application.
+     *
+     * @var array
+     */
+    protected $commands = [
+        StatsDownload::class,
+        StatsCleanup::class
+    ];
+
+    /**
+     * Define the application's command schedule.
+     *
+     * @param Schedule $schedule
+     */
+    protected function schedule(Schedule $schedule)
+    {
+        // Perform hourly stats download
+
+        $schedule->command('stats:download')
```

Comment:
This is a trivial example of scheduling the stats download to perform an hourly download, then run the cleanup task on completion.